src/mame/machine/amstrad.cpp

```diff
@@ -3230,7 +3230,7 @@ MACHINE_RESET_MEMBER(amstrad_state,kccomp)
 	amstrad_common_init();
 	kccomp_reset_machine();
 
-	/* bit 1 = /TEST. When 0, KC compact will enter data transfer
+	/* bit 1 = /TEST. When 0, KC Compact will enter data transfer
```

Comment:
This is inconsistent with the other place you changed to “KC compact”.  If you look at pictures of the machine, the label actually does say “KC compact” with a lowercase “c”.

Reply:
Correct. I can't remember why I changed only this one.
Do you need me to fix both of your remarks with a new PR ?

Reply:
Yes, please make a new PR, and give it a more descriptive title.